ci/codeowners-validator/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  fetchpatch,
+}:
+buildGoModule {
+  name = "codeowners-validator";
```

Comment:
The last release of `codeowners-validator` seems to have been in 2022. For longevity of this solution, it may be worthwhile to consider a fork, simply to ensure it gets regular updates, and we could include patches such as [the one you opened](https://github.com/mszostok/codeowners-validator/pull/222) without waiting for an unresponsive upstream

Reply:
Also it should set pname+version